bin/improver-generate-landmask-ancillary

```diff
@@ -0,0 +1,75 @@
+#   list of conditions and the following disclaimer.
+#
+# * Redistributions in binary form must reproduce the above copyright notice,
+#   this list of conditions and the following disclaimer in the documentation
+#   and/or other materials provided with the distribution.
+#
+# * Neither the name of the copyright holder nor the names of its
+#   contributors may be used to endorse or promote products derived from
+#   this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+"""Script to run landmask ancillary generation."""
+
+import argparse
+import os
+import iris
+
+from improver.generate_ancillaries.generate_ancillary import (
+    CorrectLandSeaMask)
+from improver.generate_ancillaries.generate_ancillary import (
+    find_standard_ancil)
+
+
+def main():
+    """Load in arguments and get going."""
+    parser = argparse.ArgumentParser(
+        description=('Read the model landmask, interpolate '
+                     'to the standard grid and correct '
+                     'to boolean values.'))
+    parser.add_argument('--force', dest='force', default=False,
+                        action='store_true',
+                        help=('If True, ancillaries will be generated '
+                              'even if doing so will overwrite existing '
+                              'files.'))
+    parser.add_argument('input_filepath_stage', metavar='INPUT_FILE_STAGE',
+                        help=('A path to an input NetCDF file (from StaGE) '
+                              'to be processed.'))
+    parser.add_argument('output_filepath', metavar='OUTPUT_FILE',
+                        help='The output path for the processed NetCDF')
+    parser.add_argument('grid', metavar='STANDARD_GRID',
+                        help='The standard grid to process landmask on.')
+    args = parser.parse_args()
+
+    # Check if improver ancillary already exists.
+    if not os.path.exists(args.output_filepath) or args.force:
+        landmask = find_standard_ancil(
+            args.grid, args.input_filepath_stage)
+        landmask = CorrectLandSeaMask().process(landmask)
+        iris.save(landmask, args.output_filepath, unlimited_dimensions=[])
```

Comment:
Should the resulting diagnostic be renamed land_binary_mask as it is no longer a fractional land area?